services/slam/builtin/builtin_test.go

```diff
@@ -966,46 +998,89 @@ func TestSLAMProcessSuccess(t *testing.T) {
 
 	createFakeSLAMLibraries()
 
-	attrCfg := &builtin.AttrConfig{
-		Sensors:          []string{"good_color_camera"},
-		ConfigParams:     map[string]string{"mode": "mono", "test_param": "viam"},
-		DataDirectory:    name,
-		MapRateSec:       &validMapRate,
-		DataRateMs:       validDataRateMS,
-		InputFilePattern: "10:200:1",
-		Port:             "localhost:4445",
-	}
-
-	// Create slam service
 	logger := golog.NewTestLogger(t)
-	grpcServer := setupTestGRPCServer(attrCfg.Port)
-	svc, err := createSLAMService(t, attrCfg, "fake_orbslamv3", logger, false, true)
-	test.That(t, err, test.ShouldBeNil)
 
-	slamSvc := svc.(internal.Service)
-	processCfg := slamSvc.GetSLAMProcessConfig()
-	cmd := append([]string{processCfg.Name}, processCfg.Args...)
-
-	cmdResult := [][]string{
-		{slam.SLAMLibraries["fake_orbslamv3"].BinaryLocation},
-		{"-sensors=good_color_camera"},
-		{"-config_param={mode=mono,test_param=viam}", "-config_param={test_param=viam,mode=mono}"},
-		{"-data_rate_ms=200"},
-		{"-map_rate_sec=200"},
-		{"-data_dir=" + name},
-		{"-input_file_pattern=10:200:1"},
-		{"-port=localhost:4445"},
-		{"--aix-auto-update"},
-	}
+	t.Run("Test online SLAM process with default parameters", func(t *testing.T) {
 
-	for i, s := range cmd {
-		t.Run(fmt.Sprintf("Test command argument %v at index %v", s, i), func(t *testing.T) {
-			test.That(t, s, test.ShouldBeIn, cmdResult[i])
-		})
-	}
```

Comment:
[q] Why was this test removed?

Reply:
Also the reason those tests were removed was because they originally existed to test the delete_processed_data parameter. Now that we done that separately, having those tests is a waste (basically this test is only checking that go and turn a bool into a string). The other parameters that were also being varied will undergo testing similar to delete_processed_data once we split out their Determine functions.

As of now I just reverted us back to having two success offline/online and one failure of the actual SLAM process and focused on the default parameters. 

Happy to change it back if you'd like

Reply:
No I think that's fine!